hackathon_site/registration/models.py

```diff
@@ -75,6 +77,9 @@ class Application(models.Model):
             )
         ],
     )
+    email = models.EmailField(max_length=254)
+    city = models.CharField(max_length=255, null=True, blank=True)
```

Comment:
here null and blank are true, did we confirm with logistics that these fields are optional?